hollow/src/main/java/com/netflix/hollow/api/producer/enforcer/AbstractSingleProducerEnforcer.java

```diff
@@ -34,6 +34,8 @@
 
     protected abstract boolean _isPrimary();
 
+    protected abstract void _force();
```

Comment:
Let's implement this to throw an UnsupportedOperationException by default? This way we won't break existing clients?